frontend/src/app/main/ui/ds/product/cta.scss

```diff
@@ -0,0 +1,36 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) KALEIDOS INC
+
+@use "../colors.scss" as *;
+@use "../_sizes.scss" as *;
+@use "../_borders.scss" as *;
+@use "../typography.scss" as t;
+
+:global(.light) {
+  --color-cta-background: #{$purple-100};
+  --color-cta-border: #{$purple-600-10};
+}
+
+:global(.default) {
+  --color-cta-border: #{$green-950};
+  --color-cta-background: #{$green-1000};
+}
```

Comment:
The right way of doing this is via semantic tokens.

```scss
// colors.scss
:global(.light) {
  --whatever-design-has-called-this-token: #{$purple-100};
}

:global(.default) {
  --whatever-design-has-called-this-token: #{$green-1000};
}
```

And then, at a component level:

```scss
// cta.scss
.cta {
  --color-cta-background: var(--whatever-design-has-called-this-token);
}
```

Design is responsible of naming semantic tokens, hence `--whatever-design-has-called-this-token`. They usually ask for our input, but it's their decision and it should be added to the Foundations file.